kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,200 @@
+
+export type SubscriptionData = {
+    id: string,
+    amount: number,
+    brand: string,
+    expires_at: string,
+    last4: string,
+    zip: string,
+    next_payment_at: string,
+};
+
+type Props = {
+    locale: string,
+};
+export const title = gettext('Manage monthly subscriptions');
+export const successMsg = gettext(
+    'Your monthly subscription has been successfully canceled.'
+);
+
+const ManagementPage = ({ locale }: Props): React.Node => {
+    const [showForm, setShowForm] = React.useState<boolean>(false);
+    const [status, setStatus] = React.useState<'success' | 'error' | 'idle'>(
+        'idle'
+    );
+    const [subscription, setSubscription] = useState<?SubscriptionData>(null);
+    const [canceled, setCanceled] = useState<?boolean>(false);
+    const userData: ?UserData = useContext(UserProvider.context);
+
+    useEffect(() => {
+        if (userData && userData.isSubscriber) {
+            const handleSuccess = (data) => {
+                const [subscription] = data.subscriptions;
+                setSubscription(subscription);
+            };
+            const handleError = () => setStatus('error');
+            getSubscriptions(handleSuccess, handleError);
+        }
+    }, [userData]);
+
+    // if there's no user data yet, don't render anything
+    if (!userData) {
+        return null;
+    }
+
+    const handleClick = (event) => {
+        event.preventDefault();
+        setShowForm(true);
+    };
+
+    const handleDeleteSuccess = () => {
+        setSubscription(null);
+        setCanceled(true);
+        setStatus('success');
+    };
+
+    const renderSuccess = () => (
+        <p className="alert success" data-testid="success-msg">
+            {successMsg}
+        </p>
+    );
+
+    const renderInvitation = () => (
+        <div className="active-subscriptions">
+            <Interpolated
+                id={gettext(
+                    'You have no active subscriptions. Why not <signupLink />?'
+                )}
+                signupLink={
+                    <a href={`/${locale}/payments/`}>{gettext('set one up')}</a>
+                }
+            />
+        </div>
+    );
+
+    const renderSubscriptions = () => {
+        if (!subscription) {
+            return false;
+        }
+        const date = new Date(subscription.next_payment_at);
+        const nextPaymentDate = formatDate(
+            locale,
+            subscription.next_payment_at
+        );
+        const lastActiveDate = formatDate(
+            locale,
+            date.setDate(date.getDate() - 1)
+        );
+        return (
+            <>
+                <p>
+                    {interpolate(
+                        'Next payment occurs on %(nextPaymentDate)s.',
+                        {
+                            nextPaymentDate,
+                        }
+                    )}
+                </p>
+                <div className="active-subscriptions">
+                    <ul>
```

Comment:
Do we need this wrapped in a `div`? Could we instead simply add the `active-subscriptions` class directly to the `ul`?

Reply:
Changed-- thank you!